protocol/x/rewards/types/params.go

```diff
@@ -10,9 +10,9 @@ import (
 func DefaultParams() Params {
 	return Params{
 		// Corresponds to module account address: dydx16wrau2x4tsg033xfrrdpae6kxfn9kyuerr5jjp
-		TreasuryAccount:  "rewards_treasury",
+		TreasuryAccount:  TreasuryAccountName,
 		Denom:            "dv4tnt",
-		DenomExponent:    -6,
+		DenomExponent:    -18,
 		MarketId:         1,
```

Comment:
is this market id correct?

Reply:
Actually going to revert this back to `1` since the default state is used in multiple e2e tests.
This field is meant to be overridden in `testnet` and prod network; `1` is a default sample value so prefer to keep it for now.